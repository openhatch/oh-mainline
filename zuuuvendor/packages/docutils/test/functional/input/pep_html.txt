PEP: 100
Title: Test PEP
Version: 42
Last-Modified: A long time ago.
Author: John Doe <john@example.org>
Discussions-To: <devnull@example.org>
Status: Draft
Type: Standards Track
Content-Type: text/x-rst
Created: 01-Jun-2001
Post-History: 13-Jun-2001


Abstract
========

This is just a test [#]_.  See the `PEP repository`_ for the real
thing.

.. _PEP repository: http://www.python.org/peps/


Copyright
=========

This document has been placed in the public domain.


References and Footnotes
========================

.. [#] PEP editors: peps@python.org
