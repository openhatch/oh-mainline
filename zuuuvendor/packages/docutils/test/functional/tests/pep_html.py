# Source and destination file names.
test_source = "pep_html.txt"
test_destination = "pep_html.html"

# Keyword parameters passed to publish_file.
reader_name = "pep"
parser_name = "rst"
writer_name = "pep_html"

# Settings
settings_overrides['python_home'] = "http://www.python.org"
settings_overrides['pep_home'] = "http://www.python.org/peps"
settings_overrides['no_random'] = 1
settings_overrides['cloak_email_addresses'] = 1
# local copy of default stylesheet:
settings_overrides['stylesheet_path'] = ( 
    'functional/input/data/html4css1.css')
