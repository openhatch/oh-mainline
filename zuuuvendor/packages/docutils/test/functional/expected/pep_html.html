<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<!--
This HTML is auto-generated.  DO NOT EDIT THIS FILE!  If you are writing a new
PEP, see http://www.python.org/dev/peps/pep-0001 for instructions and links
to templates.  DO NOT USE THIS HTML FILE AS YOUR TEMPLATE!
-->
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="generator" content="Docutils 0.11: http://docutils.sourceforge.net/" />
  <title>PEP 100 -- Test PEP</title>
  <link rel="stylesheet" href="../input/data/html4css1.css" type="text/css" />
</head>
<body bgcolor="white">
<table class="navigation" cellpadding="0" cellspacing="0"
       width="100%" border="0">
<tr><td class="navicon" width="150" height="35">
<a href="http://www.python.org/" title="Python Home Page">
<img src="http://www.python.org/pics/PyBanner000.gif" alt="[Python]"
 border="0" width="150" height="35" /></a></td>
<td class="textlinks" align="left">
[<b><a href="http://www.python.org/">Python Home</a></b>]
[<b><a href="http://www.python.org/dev/peps/">PEP Index</a></b>]
[<b><a href="http://www.python.org/peps/pep-0100.txt">PEP Source</a></b>]
</td></tr></table>
<div class="document">
<table class="rfc2822 docutils field-list" frame="void" rules="none">
<col class="field-name" />
<col class="field-body" />
<tbody valign="top">
<tr class="field"><th class="field-name">PEP:</th><td class="field-body">100</td>
</tr>
<tr class="field"><th class="field-name">Title:</th><td class="field-body">Test PEP</td>
</tr>
<tr class="field"><th class="field-name">Version:</th><td class="field-body">42</td>
</tr>
<tr class="field"><th class="field-name">Last-Modified:</th><td class="field-body"><a class="reference external" href="http://svn.python.org/view/*checkout*/peps/trunk/pep-0100.txt">A long time ago.</a></td>
</tr>
<tr class="field"><th class="field-name">Author:</th><td class="field-body">John Doe &lt;john&#32;&#97;t&#32;example.org&gt;</td>
</tr>
<tr class="field"><th class="field-name">Discussions-To:</th><td class="field-body">&lt;<a class="reference external" href="mailto:devnull&#37;&#52;&#48;example&#46;org?subject=PEP%20100">devnull&#32;&#97;t&#32;example.org</a>&gt;</td>
</tr>
<tr class="field"><th class="field-name">Status:</th><td class="field-body">Draft</td>
</tr>
<tr class="field"><th class="field-name">Type:</th><td class="field-body">Standards Track</td>
</tr>
<tr class="field"><th class="field-name">Content-Type:</th><td class="field-body"><a class="reference external" href="http://www.python.org/dev/peps/pep-0012">text/x-rst</a></td>
</tr>
<tr class="field"><th class="field-name">Created:</th><td class="field-body">01-Jun-2001</td>
</tr>
<tr class="field"><th class="field-name">Post-History:</th><td class="field-body">13-Jun-2001</td>
</tr>
</tbody>
</table>
<hr />
<div class="contents topic" id="contents">
<p class="topic-title first">Contents</p>
<ul class="simple">
<li><a class="reference internal" href="#abstract" id="id5">Abstract</a></li>
<li><a class="reference internal" href="#copyright" id="id6">Copyright</a></li>
<li><a class="reference internal" href="#references-and-footnotes" id="id7">References and Footnotes</a></li>
</ul>
</div>
<div class="section" id="abstract">
<h1><a class="toc-backref" href="#id5">Abstract</a></h1>
<p>This is just a test <a class="footnote-reference" href="#id2" id="id1">[1]</a>.  See the <a class="reference external" href="http://www.python.org/peps/">PEP repository</a> <a class="footnote-reference" href="#id3" id="id4">[2]</a> for the real
thing.</p>
</div>
<div class="section" id="copyright">
<h1><a class="toc-backref" href="#id6">Copyright</a></h1>
<p>This document has been placed in the public domain.</p>
</div>
<div class="section" id="references-and-footnotes">
<h1><a class="toc-backref" href="#id7">References and Footnotes</a></h1>
<table class="docutils footnote" frame="void" id="id2" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id1">[1]</a></td><td>PEP editors: <a class="reference external" href="mailto:peps&#37;&#52;&#48;python&#46;org">peps<span>&#64;</span>python<span>&#46;</span>org</a></td></tr>
</tbody>
</table>
<table class="docutils footnote" frame="void" id="id3" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id4">[2]</a></td><td><a class="reference external" href="http://www.python.org/peps/">http://www.python.org/peps/</a></td></tr>
</tbody>
</table>
</div>

</div>
</body>
</html>
